pyproject.toml

```
@@ -33,7 +33,7 @@ classifiers = [
 dependencies = ["attrs>=17.4.0", "importlib_metadata; python_version<'3.8'"]
 
 [project.optional-dependencies]
-tests = ["pytest", "moto<4.2.1"]
+tests = ["pytest", "moto<5.1.0"]
```

```suggestion
tests = ["pytest", "moto"]
```

I've just done the spelunking – the reason the pin was there was really just because it broke the tests. Since we have no way to automatically re-check the pin, the should be none until it breaks again.

---

Yes, I agree!